theme/static/js/hs_file_browser.js

```diff
@@ -690,14 +690,15 @@ function bindFileBrowserItemEvents() {
             var fileExtension = fileName.substr(fileName.lastIndexOf("."), fileName.length);
 
             // toggle apps by file extension and aggregations
+            let hasTools = false;
             menu.find("li.btn-open-with").each(function() {
                 var agg_app = false;
-                if ($(this).attr("agg-types")){
-                    agg_app = $.inArray(fileAggType, $(this).attr("agg-types").split(",")) !== -1;
+                if ($(this).attr("data-agg-types")){
+                    agg_app = $.inArray(fileAggType, $(this).attr("data-agg-types").split(",")) !== -1;
                 }
                 var extension_app = false;
-                if ($(this).attr("file-extensions")){
-                    var extensions = $(this).attr("file-extensions").split(",");
+                if ($(this).attr("data-file-extensions")){
+                    var extensions = $(this).attr("data-file-extensions").split(",");
                     for (var i = 0; i < extensions.length; ++i) {
```

Comment:
do you want this to be var extensions or let?

Reply:
also let i=

Reply:
It's better to use `let` here. I'll make this change, but this also applies to almost any JavaScript file that still uses `var` for short-scoped variables.